bluepyentity/app/explorer.py

```diff
@@ -199,27 +200,42 @@ def leaves(adt, agg):
             self._refresh_all(url)
             return
 
-    def on_key(self, event: events.Key) -> None:
-        """Manage key pressed events."""
-        if event.character == "f":
-            self._follow_cmd = not self._follow_cmd
-            if self._follow_cmd:
-                self.display_hints()
-            else:
-                self.hide_hints()
+    BINDINGS = [
+        ("f", "follow", "Open a link"),
+        ("b", "back", "Back"),
+        ("q", "quit", "Quit"),
+    ]
+
+    async def action_follow(self) -> None:
+        """Follow link"""
+        self._follow_cmd = not self._follow_cmd
+
+        if self._follow_cmd:
+            self.display_hints()
+        else:
+            self.hide_hints()
+
+    async def action_back(self) -> None:
+        """Navigate back"""
+        if self._follow_cmd:
             return
 
-        if event.character == "b":
-            if len(self._previous_urls) > 1:
-                self._previous_urls.pop()
-                url = self._previous_urls[-1]
-                self._invalidate_root()
-                self._refresh_all(url)
+        if len(self._previous_urls) > 1:
+            self._previous_urls.pop()
+            url = self._previous_urls[-1]
+            self._invalidate_root()
+            self._refresh_all(url)
+
+    async def action_quit(self) -> None:
+        """Quit the app"""
+        if self._follow_cmd:
```

Comment:
that is a bit annoying to have that for all the other keys. but that is fine for now.